perun-base/src/main/resources/perun-roles.yml

```diff
@@ -2068,6 +2107,8 @@ perun_policies:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Member
```

Comment:
please add also check for Group and Vo here, changing the member's status can gain him access that is managed by group or VO...